config/crd/bases/infrastructure.cluster.x-k8s.io_azureclusters.yaml

```diff
@@ -487,9 +527,10 @@ spec:
                             this resource.
                           type: string
                         internalLBIPAddress:
-                          description: InternalLBIPAddress is the IP address that
+                          description: 'InternalLBIPAddress is the IP address that
```

Comment:
seems like an extra `'`

Reply:
I think that's how deprecated fields are marked? There is no `'` in the type description, but looks like cidrBlock description is also in single quotes